app_legacy/Helpers/render/achievement.php

```diff
@@ -111,7 +111,7 @@ function renderAchievementCard(int|string|array $achievement, ?string $context =
     $tooltip .= "<div><b>$title</b></div>";
     $tooltip .= "<div class='mb-1'>$description</div>";
     if ($achPoints) {
-        $tooltip .= "<div>$achPoints Points</div>";
+        $tooltip .= "<div>$achPoints " . ($achPoints == 1 ? "Point" : "Points") . "</div>";
```

Comment:
While this is perfectly fine, you could use Laravel's helper function:
```
       $tooltip .= "<div>$achPoints " . Str::plural('Point', $achPoints) . "</div>";
```
You can do it [from the lang](https://laravel.com/docs/10.x/localization#pluralization) files too, but I don't know if we're ready for that yet.

Reply:
yup, I wrote the `__res()` helper for that which will allow us to have everything translated in the future.
This should work fine anywhere: `__res('point', 1)`.

Reply:
Cool, good to know. This has been adjusted to use `__res()`.